src/gui/guiFormSpecMenu.cpp

```diff
@@ -3665,6 +3842,12 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 			}
 		}
 
+		if (event.GUIEvent.EventType == gui::EGET_SCROLL_BAR_CHANGED) {//hier
```

Comment:
// redundant comment

Reply:
Sorry, that was a marker to jump around in code.

Reply:
Done.